installer-scripts/SwiftFormat.wxs

```diff
@@ -2,42 +2,41 @@
   <Package
       Language="1033"
       Manufacturer="nicklockwood"
-      Name="SwiftFormat for Windows x86_64"
+      Name="SwiftFormat for Windows $(var.ProductArchitecture)"
```

Comment:
I don't think it matters but this is probably going to expand to `amd64` now. @compnerd do we care?

Reply:
I think that we can drop the architecture name entirely.  It adds no value and is something I'm considering for the toolchain itself: https://github.com/apple/swift-installer-scripts/pull/196

Reply:
+1! Drop 'em arches!

Reply:
Done